src/explore-education-statistics-frontend/src/services/googleAnalyticsService.ts

```diff
@@ -54,37 +53,3 @@ export function logEvent({ category, action, label, value }: AnalyticsEvent) {
     value,
   });
 }
-
-export function logOutboundLink(label: string, url: string, newTab?: boolean) {
-  // we don't want to attempt to log outbound links if the user has disabled analytics
-  // but we still want any links that are external to open in a new tab
-
-  function openLink() {
-    if (newTab) {
-      window.open(url, '_blank');
-      return;
-    }
-    document.location.href = url;
-  }
-
-  if (!initialised) {
-    openLink();
-    return;
-  }
-
-  ReactGA.outboundLink(
-    {
-      label,
-    },
-
-    function hitCallback() {
-      openLink();
-    },
-  );
-}
-
-export const logException = (description: string, fatal = false) => {
```

Comment:
Have you tried doing something such as the following to log exceptions? This is what react-ga sends under the hood I believe so it should just work 

```typescript 
export function logException(description: string, fatal = false) {
  if (initialised) {
    ReactGA.set({
      hitType: 'exception',
      exDescription: description,
      exFatal: fatal,
    });
  }
}

```

Reply:
It looks like it doesn't support it https://github.com/codler/react-ga4/blob/master/src/ga4.js#L341